lib/widget/club_detail/activity/activity_list_item.dart

```diff
@@ -1,75 +1,104 @@
-                height: ThemeDimens.padding16,
-              ),
-              Wrap(
-                direction: Axis.horizontal,
-                alignment: WrapAlignment.start,
-                children: [
-                  StackedData(
-                    label: 'Distance',
-                    value: activity.distance == null ? 'unkown' : activity.distance!.toString(),
-                  ),
-                  StackedData(
-                    label: 'Moving Time',
-                    value: activity.movingTime == null ? 'unkown' : activity.movingTime!.toString(),
-                  ),
-                  StackedData(
-                    label: 'Elapsed Time',
-                    value: activity.elapsedTime == null ? 'unkown' : activity.elapsedTime!.toString(),
-                  ),
-                  StackedData(
-                    label: 'Total Elevation Gain',
-                    value: activity.totalElevationGain == null ? 'unkown' : activity.totalElevationGain!.toString(),
-                  ),
-                ],
-              ),
-              const Spacer(),
-              Text(
-                activity.athlete == null ? '' : 'By ${activity.athlete!.firstname!} ${activity.athlete!.lastname!}',
-                style: theme.lightTextTheme.bodyUltraSmall,
+      childBuilderTheme: (context, theme) => MouseRegion(
+        cursor: SystemMouseCursors.click,
+        child: AnimatedContainer(
+          duration: ThemeDurations.shortAnimationDuration(),
+          decoration: BoxDecoration(
+            color: isSelected ? theme.colorsTheme.accent : theme.colorsTheme.accent.withOpacity(0.15),
+            borderRadius: BorderRadius.circular(ThemeDimens.largeCardBorderRadius),
+            boxShadow: [
+              BoxShadow(
+                blurRadius: 5,
+                spreadRadius: 3,
+                color: isSelected ? theme.colorsTheme.accent.withOpacity(0.2) : theme.colorsTheme.shadow,
               ),
             ],
           ),
+          child: Padding(
+            padding: const EdgeInsets.all(
+              ThemeDimens.padding16,
+            ),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  activity.name,
+                  style: theme.lightTextTheme.titleSmall,
+                  maxLines: 1,
+                  overflow: TextOverflow.ellipsis,
+                ),
+                const SizedBox(
+                  height: ThemeDimens.padding16,
+                ),
+                Row(
+                  children: [
+                    Expanded(
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          StackedData(
+                            label: localization.clubDetailTotalDistanceTitle,
+                            value: '${(activity.distance / 1000).toStringAsFixed(2)} Km',
```

Comment:
Use formatters instead of hardcodedl ike this.. What if I have an activity wit 1 meter? Would not look nice..